client/src/app/shared/rest/rest-extractor.service.ts

```diff
@@ -45,8 +45,18 @@ export class RestExtractor {
       errorMessage = err.error.message
       console.error('An error occurred:', errorMessage)
     } else if (err.status !== undefined) {
-      const body = err.error
-      errorMessage = body ? body.error : 'Unknown error.'
+      // A server-side error occurred.
+      let body = JSON.parse(err.error)
```

Comment:
Add a "TODO: remove when https://github.com/angular/angular/pull/19477#issuecomment-335795151 is fixed"